src/main/java/frc/robot/commands/SetShooterVel.java

```diff
@@ -42,6 +45,6 @@ public void end(boolean interrupted) {
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    return (pivot.getCurrentPositionInRad() >= targetPositionRotations * 0.98);
+    return ((shooter.getTargetBottomVelocityInMPS() >= targetBottomVel * 0.98) && (shooter.getTargetTopVelocityInMPS() >= targetTopVel *0.98));
```

Comment:
is this command actually used?
we changed another similar command (runShooter?) to make sure we hit the velocity on both rollers consistently (vs just hitting it 1x)